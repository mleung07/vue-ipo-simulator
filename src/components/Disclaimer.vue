<template>
  <div class="footer">
    <div class="disclaimer">
      Disclaimer: The calculation is for reference only.
    </div>
  </div>
</template>

<script>
export default {
  name: "Disclaimer",
};
</script>

<style lang="scss" scooped>
.footer {
  padding: 16px;
  .disclaimer {
    font-size: 12px;
    color: #808080;
    text-align: center;
  }
}
</style>
