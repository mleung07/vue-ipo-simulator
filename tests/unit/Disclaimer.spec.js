import { shallowMount, mount } from "@vue/test-utils";
import Disclaimer from "@/components/Disclaimer.vue";

describe("Disclaimer.vue", () => {
  it("renders footer", () => {
    const wrapper = shallowMount(Disclaimer);
    expect(wrapper.text()).toMatch(
      "Disclaimer: The calculation is for reference only."
    );
  });

  it("matches snapshot", () => {
    const wrapper = mount(Disclaimer);
    expect(wrapper.element).toMatchSnapshot();
  });
});
