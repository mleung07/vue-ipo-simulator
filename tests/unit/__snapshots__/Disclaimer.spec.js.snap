// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer.vue matches snapshot 1`] = `
<div
  class="footer"
>
  <div
    class="disclaimer"
  >
    
    Disclaimer: The calculation is for reference only.
  
  </div>
</div>
`;
